common/test/common_tests/types/tokens_lib_test.cljc

```diff
@@ -1241,7 +1241,8 @@
                                    "modified-at" now
                                    "name" "theme-1"
                                    "selectedTokenSets" {"core" "enabled"}}]
-                       "$metadata" {"tokenSetOrder" ["core"]}
+                       "$metadata" {"tokenSetOrder" ["core"]
+                                    "activeSets" #{},  "activeThemes" #{}}
                        "core"
                        {"colors" {"red" {"600" {"$value" "#e53e3e"
                                                 "$type" "color"
```

Comment:
Perhaps you could add some more tests to encode/decode files with active sets, both with or without non hidden themes.